src/app/layout.tsx

```diff
@@ -16,7 +19,10 @@ export default function RootLayout({
 }>) {
   return (
     <html lang="en">
+      <Navbar/>
       <body className={inter.className}>{children}</body>
```

Comment:
Try to put the NavBar within the body tag

```suggestion
      <body className={inter.className}>
          <Navbar/>
          {children}
      </body>
```

Reply:
It should solve some conflicts